README.md

```diff
@@ -42,20 +46,42 @@ Usage: esp-generate [OPTIONS] --chip <CHIP> <NAME>
 
 Arguments:
   <NAME>
+          Name of the project to generate
 
 Options:
-  -c, --chip <CHIP>      [possible values: esp32, esp32s2, esp32s3, esp32c2, esp32c3, esp32c6, esp32h2]
+  -c, --chip <CHIP>
+          Chip to target
```

Comment:
Maybe we should just remove this output? I don't have a lot of confidence in it being kept up to date, I think the quickstart section above does a good enough job.

Reply:
Yeah fair enough, was just blindly updating things but I agree